src/openrct2-ui/title/TitleSequencePlayer.cpp

```diff
@@ -406,26 +406,27 @@ namespace OpenRCT2::Title
 
         void CloseParkSpecificWindows()
         {
-            WindowCloseByClass(WindowClass::ConstructRide);
-            WindowCloseByClass(WindowClass::DemolishRidePrompt);
-            WindowCloseByClass(WindowClass::EditorInventionListDrag);
-            WindowCloseByClass(WindowClass::EditorInventionList);
-            WindowCloseByClass(WindowClass::EditorObjectSelection);
-            WindowCloseByClass(WindowClass::EditorObjectiveOptions);
-            WindowCloseByClass(WindowClass::EditorScenarioOptions);
-            WindowCloseByClass(WindowClass::Finances);
-            WindowCloseByClass(WindowClass::FirePrompt);
-            WindowCloseByClass(WindowClass::GuestList);
-            WindowCloseByClass(WindowClass::InstallTrack);
-            WindowCloseByClass(WindowClass::Peep);
-            WindowCloseByClass(WindowClass::Ride);
-            WindowCloseByClass(WindowClass::RideConstruction);
-            WindowCloseByClass(WindowClass::RideList);
-            WindowCloseByClass(WindowClass::Scenery);
-            WindowCloseByClass(WindowClass::Staff);
-            WindowCloseByClass(WindowClass::TrackDeletePrompt);
-            WindowCloseByClass(WindowClass::TrackDesignList);
-            WindowCloseByClass(WindowClass::TrackDesignPlace);
+            auto* windowMgr = Ui::GetWindowManager();
```

Comment:
Should this be in here, or is this also something better handled by the window manager?